apps/io-services-cms-webapp/src/watchers/__tests__/on-services-lifecycles-change.test.ts

```diff
@@ -24,12 +24,33 @@ const aService = {
   },
 } as unknown as ServiceLifecycle.ItemType;
 
+const aPublicationService = {
```

Comment:
Try to avoid service complete duplication using the spread operator, for example:
`const aPublicationService = {...aService, max_allowed_payment_amount: 1000000}`

Reply:
i tried but it doesn't work like that , it adds max_allowed_payment_amount like a new parameter at the end of the service , and does not change the one inside service.data.max_allowed_payment_amount